gradle/dependencies.gradle

```diff
@@ -66,6 +66,7 @@ versions += [
   hamcrest: "2.1",
   log4j: "1.2.17",
   scalaLogging: "3.9.0",
+  japicmp: "0.13.0",
```

Comment:
Why is there not a corresponding `libs` line for this?

Reply:
I noticed it was the convention for other buildscript dependencies too. Should I fix it for the other buildscript dependencies as well?